client/me/billing-history/receipt.jsx

```diff
@@ -177,11 +178,13 @@ class BillingReceipt extends React.Component {
 		const groupedTransactionItems = groupDomainProducts( transaction.items, translate );
 
 		const items = groupedTransactionItems.map( item => {
+			const termLabel = getPlanTermLabel( item.wpcom_product_slug, translate );
 			return (
 				<tr key={ item.id }>
 					<td className="billing-history__receipt-item-name">
 						<span>{ item.variation }</span>
 						<small>({ item.type_localized })</small>
+						<em>{ termLabel }</em>
```

Comment:
It's unlikely to happen, but `termLabel` still can be `null` given the behavior of `getPlanTermLabel()`. Could we add a null check for `termLabel` just like we've done in `serviceNameDescription()` function below?